src/client/containers/EditBoardPage/EditBoardPage.container.js

```diff
@@ -1,5 +1,115 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useUser } from '../../firebase/UserContext';
+import './EditBoardPage.styles.css';
+import PropTypes from 'prop-types';
+import ArrowButton from '../../components/ArrowButton/ArrowButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import Input from '../../components/InputComponent/InputComponent';
+import Dropzone from '../DropZone/Dropzone.container';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+import { ApiError } from '../../ErrorBoundary';
 
-export const EditedBoard = () => (
-  <div className="EditedBoard ">The Edited Board!</div>
-);
+const EditedBoard = ({ boardInfo }) => {
+  const [name, setName] = useState(boardInfo.title);
+  const [date, setDate] = useState(boardInfo.deadline);
+  // eslint-disable-next-line no-unused-vars
+  const [error, setError] = useState(null);
+  const boardId = boardInfo.id;
+  const history = useHistory();
+  const { user } = useUser();
+  const userId = user[0].id;
+
+  const API_URL = `/api/users/${userId}/boards/${boardId}`;
```

Comment:
The board id comes from the useParams here

Reply:
@senner007 
Hi  Niels 
I have pushed my latest changes

Reply:
I'll have a look

Reply:
@senner007 
Hi Niels 
Thanks for your help.
I have added the changes
can you check?